File: mpp/lab3 2/lab3/src/main/java/org/example/domain/Bilet.java
package org.example.domain;

import java.util.UUID;

public class Bilet extends Entity <Long>{
    private String numeCumparator;
    private int numarLocSelectate;
    private long idSpectacol;

    public Bilet(Long id, String numeCumparator, int numarLocSelectate, long idSpectacol) {
        super(id);
        this.numeCumparator = numeCumparator;
        this.numarLocSelectate = numarLocSelectate;
        this.idSpectacol = idSpectacol;
    }

    public String getNumeCumparator() {
        return numeCumparator;
    }

    public void setNumeCumparator(String numeCumparator) {
        this.numeCumparator = numeCumparator;
    }

    public int getNumarLocSelectate() {
        return numarLocSelectate;
    }

    public void setNumarLocSelectate(int numarLocSelectate) {
        this.numarLocSelectate = numarLocSelectate;
    }

    public long getIdSpectacol() {
        return idSpectacol;
    }

    public void setIdSpectacol(long idSpectacol) {
        this.idSpectacol = idSpectacol;
    }


}

File: mpp/Spectacole/domain/Bilet.cs
namespace lab_mpp3.domain
{
    public class Bilet : Entity<long>
    {
        private long idSpectacol;
        private string numeCumparator;
        private int numarLocSelectate;

        public Bilet()
        {
            idSpectacol = 0;
            numeCumparator = "";
            numarLocSelectate = 0;
        }

        public Bilet(long idSpectacol, string numeCumparator, int numarLocSelectate)
        {
            this.idSpectacol = idSpectacol;
            this.numeCumparator = numeCumparator;
            this.numarLocSelectate = numarLocSelectate;
        }

        public long IdSpectacol
        {
            get { return idSpectacol; }
            set { idSpectacol = value; }
        }

        public string NumeCumparator
        {
            get { return numeCumparator; }
            set { numeCumparator = value; }
        }

        public int NumarLocSelectate
        {
            get { return numarLocSelectate; }
            set { numarLocSelectate = value; }
        }

        public override string ToString()
        {
            return $"ID Spectacol: {idSpectacol}, Nume Cumparator: {numeCumparator}, Numar Locuri Selectate: {numarLocSelectate}";
        }
    }
}